build/scripts/test-nuget-cache.ps1

```diff
@@ -0,0 +1,71 @@
+<#
+
+This script exists to help us track down the NuGet cache corruption bug that is showing up 
+in Jenkins.  It will look for critical files in the NuGet which have been zerod out.
+
+https://github.com/dotnet/roslyn/issues/19882
+
+#>
+
+[CmdletBinding(PositionalBinding=$false)]
+param ()
+
+Set-StrictMode -version 2.0
+$ErrorActionPreference="Stop"
+
+function Test-ZeroFile([string]$filePath) {
+    $bytes = [IO.File]::ReadAllBytes($filePath)
+    if ($bytes.Length -eq 0) {
+        return $false
+    }
+
+    foreach ($b in $bytes) { 
+        if ($b -ne 0) {
+            return $false
+        }
+    }
+
+    return $true
+}
+
+function Go() {
+    Push-Location (Get-PackagesDir)
+    try {
+        $failed = @()
+        foreach ($filePath in Get-ChildItem -re -in *.nuspec,*proj,*settings,*targets) {
```

Comment:
Perf. Takes a decent amount of time to do all files.  Thought we'd start with the ones we know are failing.  If it's not enough we can expand.